packages/types/lib/plans/db.ts

```diff
@@ -0,0 +1,38 @@
+import type { Timestamps } from '../db';
+
+export interface DBPlan extends Timestamps {
+    id: number;
+    account_id: number;
+    name: string;
+    trial_start_at: Date | null;
+    trial_end_at: Date | null;
+    trial_extension_count: number;
+    trial_end_notified_at: Date | null;
+
+    /**
+     * Limit the number of connections with active scripts
+     * Set to null to remove limit
+     * @default 3
+     */
+    max_connection_with_scripts: number | null;
+
+    /**
+     * Limit the number of environments that can be created
+     * Set to null to remove limit
+     * @default 2
+     */
+    max_environments: number | null;
```

Comment:
why null if it has a default value?

Reply:
because it's how postgres works, and as stated in the comment can be set to null to remove limitation

Reply:
up to you but I will personally make it not null and the code set a high enough value. Even if from a commercial point of view it is "unlimited" we should probably always have a ceiling value, just to make sure there is no accidental addition of 1000s of environments. For now  accident are unlikely because adding an environment is done via the UI but if one day we allow to do it via the api/node client it is gonna happen

Reply:
true